client/src/components/NameGenderCategoryPrice.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components';
+import NameGenderCategory from './NameGenderCategory.jsx';
+import Price from './Price.jsx';
+
+const NameGenderCategoryPrice = styled.div`
+  display: flex;
+  flex-flow: row;
+  justify-content: space-between;
+  background-Color: azure;
+`;
+
+export default ({ name, gender, category, price }) => {
+
+  return (
+    <NameGenderCategoryPrice>
```

Comment:
I'm confused by the <NameGenderCategory> and <GenderCategory> components. They seem to be redundant to me and exist only as wrappers for the category, gender, name components. 

Maybe this makes more sense in the styled components framework.

Why not something like this?

    <HeaderInfo>
         <Name>
        <Category>
        <Price>
    </HeaderInfo>

Significantly fewer components, and I think more readable as well. The components can be styled using CSS in this file.

Reply:
Here's what it looks like rendered on the Nike page.

As you can see, there's only 2 levels of nesting. In general, I think that less nesting > more nesting.

<img width="649" alt="Screen Shot 2020-10-08 at 9 33 09 PM" src="https://user-images.githubusercontent.com/29583177/95543695-1a971300-09ae-11eb-8b7a-2bfef4d86cb9.png">

Reply:
Love that feedback!  Thanks! Just what I needed.

Reply:
Glad this was helpful! 

Also, it seems some of the text disappeared in my comments, probably because I was using "<" and ">" symbols to represent the React JSX components. Sorry about that!